Scripts/Runtime/Entities/Tasks/Attributes/ResolveChannelAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = true)]
+    public class ResolveChannelAttribute : Attribute
```

Comment:
As discussed think about changing the use of the term channel to something else.
 - ResolveStream? ResolveTarget? - I thinkI like target best because it implies writing
 - Make attribute name more fluent ResolveTargetFor()

Reply:
Cool, I like `ResolveTarget` as well. Renamed. 

I think the attribute still makes sense as:

```csharp
[ResolveTarget(TimerInstance.ResolveTarget.Completed)]
public readonly TaskStream<TimerCompletedInstance> TimerCompletedDataStream;
```

Because it's saying the `TimerCompletedDataStream` is a ResolveTarget for the enum.